### ISSUE TITLE
greenkeeper enabled?

### ISSUE THREAD

**Issue by AdriVanHoudt** [greenkeeperio/greenkeeper#355]
So I manually closed the initial PR and now I am not sure whether greenkeeper is enabled.
See https://github.com/Salesflare/hapi-routes/pull/51

**Comment by finnp** [comment]
Hey @AdriVanHoudt,

no it is not enabled. We only enable the repository once the PR is considered merged on GitHub, not if it is closed. Currently we have no alternative way to enabling, so you would need to deinstall and reinstall the integration (or remove and add that repository from the integration) if you want to enable it. Then it should be enabled automatically since the dependencies are already up-to-date.

Sorry for the inconvenience. I will flag this issue as a bug.

Best,
Finn

**Comment by AdriVanHoudt** [comment]
thanks for letting me know how to enable it, is there a way I can check on what repos it is fully enabled?

**Comment by AdriVanHoudt** [comment]
btw the reason I did not merge it was https://github.com/greenkeeperio/greenkeeper/issues/64

**Comment by finnp** [comment]
Yeah via this image: `https://badges.greenkeeper.io/Salesflare/hapi-routes.svg`. Looks enabled now for you :)

Your badge: ![badge](https://badges.greenkeeper.io/Salesflare/hapi-routes.svg)

**Comment by AdriVanHoudt** [comment]
Ok but what if I want an overview of all my repos?

**Comment by finnp** [comment]
That's currently not possible.

**Comment by AdriVanHoudt** [comment]
You have a feature request then 😉

**Comment by dunnkers** [comment]
I'm currently experiencing the same issue. I closed the initial PR, but upon re-opening and merging Greenkeeper does not seem enabled, see: https://github.com/dunnkers/ember-polymer/pull/54

I re-installed the integration a couple of minutes ago, but no initial PR was sent yet. Will let know if any change.

**Comment by dunnkers** [comment]
Sweet! Greenkeeper just created a new initial PR, see [dunnkers/ember-polymer/tree/greenkeeper/initial](https://github.com/dunnkers/ember-polymer/tree/greenkeeper/initial). Re-installing the integration is not the most elegant solution, however.

**Comment by dunnkers** [comment]
My issue has been resolved 👌🏻 Greenkeeper appeared to have over-eagerly upgraded a QUnit package's major semver version, causing no tests to run and thus timing out Travis.

This issue was resolved for me by re-installing the Greenkeeper integration. Of course, however, this is not the most elegant solution. 🌺

**Comment by sanderploegsma** [comment]
I'm having the same problem: due to many (50+) github repositories containing NPM packages and node.js applications we have set the Greenkeeper plugin to 'all repositories', but when we create a new repository and Greenkeeper hits us before we can hook up Travis, we end up with a broken state.

I tried disabling and re-enabling Greenkeeper to force add a repository, but that ended up in almost 100 Pull Requests on other repos that already had a working Greenkeeper integration 😢

**Comment by Realtin** [comment]
@sanderploegsma We are currently working on making this easier. First step is that Greenkeeper support will be able to remotely 'reset' Greenkeeper on a repository with the help of this new job: https://github.com/greenkeeperio/greenkeeper/pull/479 
Then you don't have to disable and re-enable Greenkeeper for everything anymore.

**Comment by krzkaczor** [comment]
@Realtin what are the repercussions of disabling and re-enabling greenkeeper integration for whole organization (there are many other projects with GK already working)? We have one repo that is "broken" (greenkeeper branch was closed). What should we do in this case?

**Comment by Realtin** [comment]
@krzkaczor tell me the name of the repo, I can reset it remotely! (since https://github.com/greenkeeperio/greenkeeper/pull/479 is now merged)

**Comment by krzkaczor** [comment]
Woah, @Realtin this sound both great and scary :D This is the repo: https://github.com/Neufund/commit.neufund.org travis currently setup so I hope it will work this time. 

Thank you.

**Comment by Realtin** [comment]
@krzkaczor done! Soon we'll integrate the reset functionality in the https://account.greenkeeper.io/ page so everyone can do it themselves! :)
